src/apps/content-editor/src/app/components/Editor/Field/Field.js

```diff
@@ -192,7 +192,15 @@ export default connect(state => {
       return (
         <FieldTypeText
           name={name}
-          label={label}
+          label={
+            <>
```

Comment:
Lets abstract this into a `<FieldLabel>` component just within this same file. This is just about creating the habit and expectation that we componentize whenever possible.